src/dyn/qtag/qtag.cpp

```diff
@@ -593,7 +593,12 @@ complex<double> BATe(int i, int j,
 
 CMATRIX qtag_potential(MATRIX& q1, MATRIX& p1, MATRIX& s1, MATRIX& alp1, int n1, vector<int>& traj_on_surf_n1,
                        MATRIX& q2, MATRIX& p2, MATRIX& s2, MATRIX& alp2, int n2, vector<int>& traj_on_surf_n2,
-                       nHamiltonian& ham, int method, double AA, double BB, double CC){
+                       nHamiltonian& ham, int method, double ABC[3]){

```

Comment:
Boost::Python can not accept more than 16 args, so I collable three last into one. With simple code it works: https://godbolt.org/z/7qPxWEdbf

Reply:
there is a macro - called something like ARITY - which enables lifting this restriction

Reply:
Hmmm... But somewhat it did not work for me